init/src/init.ts

```diff
@@ -371,10 +371,8 @@ const exampleLoggerMiddleware = define.middleware((ctx) => {
 });
 app.use(exampleLoggerMiddleware);
 
-await fsRoutes(app, {
-  loadIsland: (path) => import(\`./islands/\${path}\`),
-  loadRoute: (path) => import(\`./routes/\${path}\`),
-});
+// Include file-system based routes here
+app.fsRoutes();
 
 if (import.meta.main) {
   await app.listen();
```

Comment:
Since it's now intended to use `deno serve -A _fresh/server.js` and not `deno run -A main.ts`, should this entrypoint code not also be removed?